### ISSUE TITLE
Fix release process

### ISSUE THREAD

**Issue by dhoepelman** [konform-kt/konform#105]
I recently took over maintainership. My current highest priority issue is to ensure I can do releases. 

I tried to automatically do them from the CI when a github release/tag was created in #92 and tried to release [0.5.0](https://github.com/konform-kt/konform/releases/tag/v0.5.0).
This seems to almost work, but sonatype rejects the release because there's some issues with the POM. Still need to look into these and fix the gradle build so it emits a valid artifact confirming to sonatype/nexus central rules.

**Comment by dhoepelman** [comment]
Issues:

```
<div class="nx-staging-repository-ActivityView-messagePanel-head"><div class="nx-staging-repository-ActivityView-messagePanel-title">Event: Failed: POM Validation</div><div class="nx-staging-repository-ActivityView-messagePanel-timestamp">Friday, May 10, 2024 16:34:46 CEST (GMT+0200)</div></div><div class="nx-staging-repository-ActivityView-messagePanel-body"><div class="nx-staging-repository-ActivityView-messagePanel-properties">
typeId | pom-staging
-- | --
failureMessage | Invalid  POM: /io/konform/konform/0.5.1/konform-0.5.1.pom: Project name missing,  Project description missing, Project URL missing, License information  missing, SCM URL missing, Developer information missing
failureMessage | Invalid  POM: /io/konform/konform-jvm/0.5.1/konform-jvm-0.5.1.pom: Project name  missing, Project description missing, Project URL missing, License  information missing, SCM URL missing, Developer information missing
failureMessage | Invalid  POM: /io/konform/konform-js/0.5.1/konform-js-0.5.1.pom: Project name  missing, Project description missing, Project URL missing, License  information missing, SCM URL missing, Developer information missing

</div></div>
```

```
<div class="nx-staging-repository-ActivityView-messagePanel-head"><div class="nx-staging-repository-ActivityView-messagePanel-title">Event: Failed: Javadoc Validation</div><div class="nx-staging-repository-ActivityView-messagePanel-timestamp">Friday, May 10, 2024 16:34:48 CEST (GMT+0200)</div></div><div class="nx-staging-repository-ActivityView-messagePanel-body"><div class="nx-staging-repository-ActivityView-messagePanel-properties">
typeId | javadoc-staging
-- | --
failureMessage | Missing: no javadoc jar found in folder '/io/konform/konform-jvm/0.5.1'

</div></div>
```

```
<div class="nx-staging-repository-ActivityView-messagePanel-head"><div class="nx-staging-repository-ActivityView-messagePanel-title">Event: Failed: Signature Validation</div><div class="nx-staging-repository-ActivityView-messagePanel-timestamp">Friday, May 10, 2024 16:36:04 CEST (GMT+0200)</div></div><div class="nx-staging-repository-ActivityView-messagePanel-body"><div class="nx-staging-repository-ActivityView-messagePanel-properties">
typeId | signature-staging
-- | --
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://keyserver.ubuntu.com:11371/">http://keyserver.ubuntu.com:11371/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="https://keys.openpgp.org/">https://keys.openpgp.org/</a>.  Upload your public key and try the operation again.
failureMessage | No  public key: Key with id: (e7840075487bf155) was not able to be located  on <a  href="http://pgp.mit.edu:11371/">http://pgp.mit.edu:11371/</a>.  Upload your public key and try the operation again.

</div></div>
```

**Comment by dhoepelman** [comment]
gpg key should be fixed

**Comment by dhoepelman** [comment]
The other issues are hopefully fixed by #108 and #109

**Comment by dhoepelman** [comment]
Release seems to have gone through 🎉
